contracts/ConnextModule.sol

```diff
@@ -3,9 +3,9 @@ pragma solidity ^0.8.16;
 
 import "@gnosis.pm/zodiac/contracts/core/Module.sol";
 import "./interfaces/IExecutor.sol";
-import {LibCrossDomainProperty} from "./lib/LibCrossDomainProperty.sol";
```

Comment:
we should make sure this file (and typed men view) are removed -- they are not necessary (unless wanted for some other reason)

Reply:
removing IExecutor as well